src/languages/en.json

```diff
@@ -0,0 +1,179 @@
+  "onboard.setupSharing.allSet": "I'm all set!",
+  "onboard.setupYourTests.title": "Setup your tests!",
+  "onboard.setupYourTests.text1": "Your tests are setup to run daily by default.\nIf you’d like to turn this off, click the {iconClock} button!",
+  "onboard.setupYourTests.text2": "To learn more about each test, click the {iconInfo} button.",
+  "onboard.setupYourTests.text3": "To see the results of your first batch of tests, click {goToMyDashboard}",
+  "onboard.setupYourTests.text3.goToMyDashboard": "Go to my dashboard!",
+  "onboard.setupYourTests.goToDashboard": "Go to my dashboard!",
+  "onboard.understandTheLaws.title": "Potential risks",
+  "onboard.understandTheLaws.text1": "Anyone monitoring your internet activity (e.g. ISP, government, employer) can know that you are running ooniprobe (even though OONI tries to make this hard).\n\nWhen testing websites for censorship, you will connect to and download data from provocative or objectionable sites (e.g. pornography) that might be illegal in your country.\n\nOONI’s HTTP invalid request line test might trigger the suspicion of your ISP and could be viewed as a form of “hacking”.",
+  "onboard.understandTheLaws.text2": "Unless you configure your settings otherwise, all data collected from your tests will by default be published by OONI and might include some personally- identifiable information (though OONI tries to prevent this from happening). The use of ooniprobe could be viewed as a form of espionage, regardless of the laws in your country. To our knowledge, no ooniprobe user has ever faced consequences from the use of our platform. However, many countries have a lengthy history of targeting and abusing digital rights activists, which could leave ooniprobe users subject to severe civil, criminal, or extra-judicial penalties. {learnMoreHere} . We encourage you to consult with a lawyer prior to installing and running ooniprobe.",
+  "onboard.understandTheLaws.text2.learnMoreHere": "Learn more here",
+  "onboard.understandTheLaws.iUnderstand": "I understand the risk.",
+  "onboard.welcome.title": "Welcome to ooniprobe!",
+  "onboard.welcome.text1": "Ooniprobe is a free and open source platform that enables you to examine internet censorship in your network!\nYou can examine how websites are blocked, and whether censorship and/or surveillance systems are present in your network. You can also measure the speed and performance of your network.\nData collected by ooniprobe can serve as evidence of censorship events.\nSince ooniprobe is an investigatory tool, using it may pose some risks.",
+  "onboard.welcome.learnMore": "Learn more",
+  "onboard.welcome.alreadyUnderstand": "I already understand the risks, take me to my dashboard.",
+  "settings.title": "Settings"
+}
```

Comment:
Specifying the content inline. This file here is generated from the `<FormatMessages>` by means of the `bin/translate.js` script.

In theory I could also not commit this file and it would still work, however it's best to commit it so that we can setup transifex to pull in translations automatically from the source code.